### PR TITLE
Clarify KQL example in graph-match-operator.md

### PR DESCRIPTION
**Pull request by ziggymsft**
The existing KQL example is **very** hard to follow. I have updated the code to clarify the purpose of the columns and how the objects are used within the graph.

Also updated to show all attack paths and the relationships; this demonstrates how the graph works.

### CODE REVIEW

**Comment by ziggymsft** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by ziggymsft** [comment]
Here is another example which may be helpful - I have added a compromised user, compromised workstation, and a Domain Controller into the example. Helps illustrate the graph and also filters on **any** type of attack against a **System**:

```
let ObjectDetails = datatable(ObjName:string, ObjType:string, ObjAge:long) 
[ 
  "Alice", "Person", 23,  
  "Bob", "Person", 31,  
  "Charlie", "Person", 32,  
  "Eve", "Person", 17,  
  "Mallory", "Person", 29,  
  "Win11-01","System",99,
  "WebApp01", "System", 99,
  "DC01", "System", 99
]; 
let Paths = datatable(SourceUser:string, DestUser:string, Relationship:string) 
[ 
  "Alice", "Bob", "communicatesWith",  
  "Alice", "WebApp01", "trusts",  
  "Bob", "WebApp01", "hasPermission",  
  "Eve", "Alice", "attack - phishing",  
  "Mallory", "Alice", "attack - malware",  
  "Mallory", "Bob", "attack - malware",
  "Eve","Charlie","attack - phishing",
  "Charlie","Win11-01","attack - compromised user",
  "Win11-01","DC01","attack - malware"
];
//
Paths 
| make-graph SourceUser --> DestUser with ObjectDetails on ObjName 
| graph-match (graph_sourceuser)-[graph_Relationship1]->(graph_TargetObject1)-[graph_Relationship2]->(graph_TargetObject2) 
  where graph_Relationship1.Relationship startswith "attack" // first step is attack
    and graph_TargetObject2.ObjType == "System"              // final target is a System
  project Attacker = graph_sourceuser.ObjName, 
          UsedMethod1 = graph_Relationship1.Relationship, 
          Compromised = graph_TargetObject1.ObjName, 
          UsedMethod2 = graph_Relationship2.Relationship, 
          TargetSystem = graph_TargetObject2.ObjName
```

**Comment by Court72** [comment]
@shsagir

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by v-dkorn** [comment]
We are working to review the changes.

**Comment by v-dkorn** [comment]
#please-close

**Comment by v-dkorn** [comment]
Relevant changes made in a separate PR. Thanks again for the contribution!

**Comment by orspod** [comment]
PR with requested changes: https://github.com/MicrosoftDocs/dataexplorer-docs-pr/pull/5132
